### ISSUE TITLE
Whether it is right about KL_loss 

### ISSUE THREAD

**Issue by yangzhikai** [junyanz/BicycleGAN#60]
I find the calculation of KL-loss  is different from VAE. 
Because:
latent_loss = 0.5 * tf.reduce_sum(　tf.square(z_mean) + tf.square(z_stddev) 
　　　　　　　　　　　　　　　　　　　　　　- tf.log(tf.square(z_stddev)) - 1,　1　)
I think the   "self.mu.pow(2) "  of line-154 in bicycle_gan_model.py  should be "self.mu.pow(-2)".

**Comment by junyanz** [comment]
@pathak22 I think it should be pow(2) according to Equation (10) in the VAE [paper](https://arxiv.org/pdf/1312.6114.pdf).
